cli/app.go

```diff
@@ -618,6 +620,134 @@ var app = &cli.App{
 						},
 					},
 				},
+				{
+					Name:      "tag",
+					Usage:     "tag binary data by filter or ids",
+					UsageText: createUsageText("data tag", nil, true),
+					Subcommands: []*cli.Command{
+						{
+							Name:  "ids",
+							Usage: "adds or removes tags from binary data by file ids for a given org and location",
+							UsageText: createUsageText("data tag ids", []string{dataFlagAdditionalTags, dataFlagOrgID,
+								dataFlagLocationID, dataFlagFileIDs, dataFlagTagCommand}, false),
+							Flags: []cli.Flag{
+								&cli.StringSliceFlag{
+									Name:     dataFlagAdditionalTags,
+									Required: true,
+									Usage:    "comma separated tags to add/remove to the data",
+								},
+								&cli.StringFlag{
+									Name:     dataFlagTagCommand,
+									Required: true,
+									Usage:    "accepted values: add, remove",
+								},
+								&cli.StringFlag{
+									Name:     dataFlagOrgID,
+									Usage:    "org ID to which data belongs",
+									Required: true,
+								},
+								&cli.StringFlag{
+									Name:     dataFlagLocationID,
+									Usage:    "location ID to which data belongs",
+									Required: true,
+								},
+								&cli.StringSliceFlag{
+									Name:     dataFlagFileIDs,
+									Usage:    "comma separated file IDs of data belonging to specified org and location",
+									Required: true,
+								},
+							},
+							Action: DataTagAction,
+						},
+						{
+							Name:      "filter",
+							Usage:     "adds or removes tags from binary data by filter",
+							UsageText: createUsageText("data tag ids", []string{dataFlagAdditionalTags, dataFlagTagCommand}, false),
+							Flags: []cli.Flag{
+								&cli.StringSliceFlag{
+									Name:     dataFlagAdditionalTags,
```

Comment:
I would expect this to be the `--tags` field and then the tags by which we are filtering to have a special name instead like `--filter-tags` or something similar. Since this is the required field and the one that is most prominently associated with the command, I would expect this to be the "tags" part of the tags command

Reply:
Ahhh -- this is a good idea! I was really struggling with the naming convention around this. Thanks!